magma/frontend/coreir_.py

```diff
@@ -18,8 +27,9 @@ def definition(cls):
 
 def DefineCircuitFromGeneratorWrapper(cirb: CoreIRBackend, namespace: str, generator: str,
                                       uniqueName: str, dependentNamespaces: list = [],
-                                      genargs: dict = {}, runGenerators = True):
+                                      genargs: dict = {}, runGenerators = True, print_error=False):
```

Comment:
Doesn't seem like this new `print_error` argument is used, can we drop this change?

Reply:
Removed `print_error`.